pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -183,34 +171,6 @@ private void consumeInputBlocks() {
     }
   }
 
-  private AggregationFunction toAggregationFunction(RexExpression aggCall, int aggregationFunctionInputRef) {
```

Comment:
@walterddr was there any reason why this was there in first place? I noticed that it was just used for `.length` calls which are no longer necessary

Reply:
no it is not used. the whole reason was i think previously i was using the agg function to do the merge. but now that we decided to only support SUM/COUNT/MIN/MAX it doesnt' necessary have to do that way)